content/desktop/kubernetes.md

```diff
@@ -68,7 +68,7 @@ You can test the command by listing the available nodes:
 $ kubectl get nodes
 
 NAME                 STATUS    ROLES     AGE       VERSION
-docker-desktop       Ready     master    3h        v1.19.7
+docker-desktop       Ready     control-plane    3h        v1.29.1
```

Comment:
We should probably re-align the column-headers as well (where needed, it's ok to slightly adjust the space in between if that makes it fit the layout better)

Reply:
Done!